src/include/kdbprivate.h

```diff
@@ -323,6 +323,11 @@ struct _KDB
 
 	Plugin * notificationPlugin; /*!< reference to global plugin for notifications.*/
 	ElektraNotificationCallbackContext * notificationCallbackContext; /*!< reference to context for notification callbacks.*/
+
+	KeySet * global; /*!< This keyset can be used by global plugins to pass data through
```

Comment:
Do we need it here if it is actually only a local variable within kdbGet/kdbSet?

Reply:
The idea was to have ksNew() only once per kdbOpen() and then ksClear() it before return such that plugins do not have to think about cleaning up.

As discussed in a meeting, part of it would be persisted in the cache, but this is for a different PR.